obsei/source/pandas_source.py

```diff
@@ -0,0 +1,103 @@
+import logging
+from obsei import configuration
+from typing import List, Optional, Dict, Any
+
+from pandas.core.frame import DataFrame
+from obsei.source.base_source import BaseSource, BaseSourceConfig
+from obsei.payload import TextPayload
+from obsei.misc.utils import (
+    DATETIME_STRING_PATTERN,
+    DEFAULT_LOOKUP_PERIOD,
+    convert_utc_time,
+)
+import pandas as pd
+
+logger = logging.getLogger(__name__)
+
+
+class PandasSourceConfig(BaseSourceConfig):
```

Comment:
Let's keep it simple and remove code related CSV/Excel.
CSV/Excel need different way of handling. 
Like reading -
- Very large file (lets say in TBs)
- Reading file from local, and remote
- Reading continuously updating file (ie CSV, Excel etc file are getting updated regularly while reading and there is need of code to handle that)

Reply:
Sure sir ,
- I'll comment the current csv and excel related code .
- For any data bigger than 1GB we would need to use something like pyspark or any other big-data platform that supports in memory computations.

Reply:
Yes exactly, future plan to incorporate it. means providing full production grade setup (data streaming and cloud hosting setup)